README.md

```diff
@@ -1,15 +1,101 @@
 # getappmap/archive-appmap
 
+> To get started with AppMap in GitHub actions, you need to start by installing the [AppMap App on the official GitHub Marketplace](https://github.com/marketplace/get-appmap)
+
+> To see a step-by-step example of how to install this action into your software project, [review the official AppMap Documentation](http://appmap.io/docs/analysis/in-github-actions).
+
 GitHub action to build and store an AppMap archive.
 
-To see a step-by-step example of how to install this action into your software project, [review the official AppMap Documentation](http://appmap.io/docs/analysis/in-github-actions).
+## Table of contents
+
+- [Prerequisites](#prerequisites)
+- [Inputs](#inputs)
+- [Examples](#examples)
+- [Development](#development)
+
+## Prerequisites
+
+The `archive-action` needs to run AFTER your test cases have executed to build and save an AppMap archive to the GitHub build asset repository. Your application will need AppMap libraries and configuration files created in the project, or you will need to use the [`install-action`](https://github.com/getappmap/install-action) to setup AppMap before your test cases run. 
+
+For more information about how to setup AppMap in your project [review the official AppMap Documentation](http://appmap.io/docs/analysis/in-github-actions).
+
+## Inputs
+
+Add a step like this to your workflow:
+
+```yaml
+- name: Archive AppMaps
+  uses: getappmap/archive-action@v1
+  with:
+    # Command working directory. Change this if your project lives in a 
+    # subdirectory or for monorepo / multi-project support
+    # Default: '.'
+    directory: /path/to/code
+
+    # 0-based numerical id of a matrix job. This id is used to store the AppMap 
+    # archive in the action cache. Once the matrix jobs complete, the `merge` 
+    # action can be used to merge the archives into a single archive, and then 
+    # save it as an artifact. If `archive-id` is set, the archive is saved to the 
+    # cache. Otherwise, it's uploaded to the artifact store. For a non-matrix 
+    # job, `archive-id` should not be used.
+    # Required for Multi-runner matrix builds.
+    archive-id: ${{ matrix.ci_node_index }}
+
+    # Revision (commit SHA) to name the AppMap archive with. The default is the 
+    # GITHUB_SHA. Under normal circumstances, the default value is correct 
+    # and this input does not need to be specified.
+    revision: 6056b0cfbacd562f4a4f274122c4a5e85542e040
+      
+    # The GitHub token to use with the GitHub API to enumerate AppMap Tools 
+    # releases.
```

Comment:
It's also used to write the "Configuration report" - we can describe all of that if we want to, or just say it's the token :-)